.gitlab-ci.yml

```
@@ -275,6 +288,28 @@ test_acceptance_ubuntu:
           --key ${RASPBIAN_NAME}/arm/${PUBLISH_NAME}
     - done
 
+.template:publish:docker-image:
+  stage: publish
+  tags:
+    - docker
+  image: docker
+  services:
+    - name: docker:20.10.8-dind
+      alias: docker
+      command: ["--mtu=1440"] # https://gitlab.com/gitlab-com/gl-infra/production/-/issues/5590
+  before_script:
+    - docker login -u $DOCKER_HUB_USERNAME -p $DOCKER_HUB_PASSWORD
+    - docker import image.tar $DOCKER_REPOSITORY:pr
```

Why not `docker load`? With load we will preserve the image tag
```suggestion
    - docker load -i image.tar
```

---

Yes, much better! I will change that.